ios/iden3CoreSDK/Iden3Identity.swift

```diff
@@ -0,0 +1,190 @@
+//
+//  Iden3Identity.swift
+//  iden3CoreSDK
+//
+//  Created by Iden3 on 12/06/2020.
+//  Copyright © 2020 Iden3. All rights reserved.
+//
+
+import Foundation
+import Iden3mobile
+
+public final class Iden3Identity {
+    
+    // MARK: - Properties
+    
+    private var identity: Iden3mobileIdentity
+    
+    // MARK: - Initializers
+       
+    init(identity: Iden3mobileIdentity) {
+        self.identity = identity
+    }
+    
+    /// Stops the Iden3 identity.
+    ///
+    public func stopIdentity() {
+        identity.stop()
+    }
+    
+    /// Request a claim from an issuer.
+    ///
+    /// - parameters:
+    ///   - issuerUrl: Url of the issuer
+    ///   - data: data needed by the issuer for requesting a claim.
+    ///   - ticketDelegate: Delegate of tickets received after requesting the claim.
+    ///
+    ///   return Iden3Ticket.
+    ///
+    public func requestClaim(issuerUrl: String, data: String, ticketDelegate: Iden3TicketDelegate?) throws -> Iden3Ticket? {
+        if (ticketDelegate == nil) {
+            do {
+                return Iden3Ticket(try identity.requestClaim(issuerUrl, data: data))
+            } catch {
+                return nil
+            }
+        } else {
+            identity.requestClaim(withCb: issuerUrl, data: data, c: nil)
+        }
+        return nil
+    }
+    
+    /// Sends a credentialValidity build from the given credentialExistance to a verifier.
```

Comment:
The credential validity is send if withZKProof is false, if it's true a zero knowlege proof is sended instead of a credential validity. Any way I think that this may be to "low level" for a mobile integrator. Maybe something more plain would be better. Example: `Send a proof of a claim to specified verifier. If withZKProof is set to true, the generated proof will hide part of the claim information to the verifier`

Reply:
done